src/main/java/org/ballerinalang/command/cmd/RemoveCommand.java

```diff
@@ -24,11 +24,10 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.PrintStream;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.util.Comparator;
 import java.util.List;
 
+import static org.ballerinalang.command.util.OSUtils.deleteFiles;
```

Comment:
Instead of import static, Can't we just call OSUtils.deleteFiles()  inside the method